src/react/components/pages/editorPage/editorSideBar.tsx

```diff
@@ -13,14 +13,14 @@ export interface IEditorSideBarProps {
     assets: IAsset[];
     onAssetSelected: (asset: IAsset) => void;
     selectedAsset?: IAsset;
+    onAssetError?: () => void;
```

Comment:
I don't feel that an individual asset error warrants bubbling this up - Let is control its display rendering internally.